book/src/drivers.md

```diff
@@ -28,6 +28,8 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [BizFlyCloud Block Storage](https://github.com/bizflycloud/csi-bizflycloud) | `volume.csi.bizflycloud.vn` | v1.2 | A Container Storage Interface (CSI) Driver for BizFly Cloud block storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion
 [CephFS](https://github.com/ceph/ceph-csi) | `cephfs.csi.ceph.com` | v0.3, >=v1.0.0 | A Container Storage Interface (CSI) Driver for CephFS | Persistent | Read/Write Multiple Pods | Yes | Expansion, Snapshot, Cloning
 [Ceph RBD](https://github.com/ceph/ceph-csi) | `rbd.csi.ceph.com` | v0.3, >=v1.0.0 | A Container Storage Interface (CSI)  Driver for Ceph RBD | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Topology, Cloning,In-tree plugin migration
+[cert-manager csi-driver](https://cert-manager.io/docs/usage/csi-driver/) | `csi.cert-manager.io` | v1.x | Uses [cert-manager](https://cert-manager.io) to provision secretless X.509 certificates for pods via ephemeral CSI volumes | Ephemeral | Read/Write Single Pod | Yes |
```

Comment:
Can you enter v1.9?
https://github.com/cert-manager/csi-lib/blob/main/go.mod#L7